inventree_brother/brother_plugin.py

```diff
@@ -120,6 +145,11 @@ def print_label(self, **kwargs):
         # ^ currently this width and height are those of the label template (before conversion to PDF
         # and PNG) and are of little use
 
+        # Printing options requires a modern-ish InvenTree backend,
+        # which supports the 'printing_options' keyword argument
+        options = kwargs.get('printing_options', {})
+        n_copies = min(1, options.get('copies', 1))
```

Comment:
Why this min function? Doesn't that mean if I set copies to 10, I still get 1? I think the min/max could be generally left out, because you already set the default at the .get function.